articles/cognitive-services/Computer-vision/includes/quickstarts-sdk/node-sdk.md

```diff
@@ -134,7 +134,7 @@ The following code detects common objects in the image and prints them to the co
 
 [!code-javascript[](~/cognitive-services-quickstart-code/javascript/ComputerVision/ComputerVisionQuickstart.js?name=snippet_objects)]
 
-Define the helper function `formatRectObjects`:
+Define the helper function `formatRectObjects` to return coordinate of top-left side, height and width of bounding box:
```

Comment:
to return the top, left, bottom, and right coordinates, along with the width and height.

Reply:
@anwesh-b please use the above text instead

Reply:
@anwesh-b apologies, I didn't see your message. Are you sure that's the case? I'm looking at the method and it seems to return each one.